### PR TITLE
Add a section addressing customization and obfuscation.

### PR DESCRIPTION
**Pull request by jvlcek**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1573511

I've done 3 improvements with this PR

1. Fixed the section hot links providing hot links from the summary at the top of the document to the relevant section in the document.
2. Added a customizing the generated configuration
3. Addressed BZ 1573511 by directing the user to sss_obfuscate to address the sited issue

### CODE REVIEW

**Comment by jvlcek** [comment]
@abellotti and @cybette Please review.

Thank you! JoeV

File: site/_posts/2017-09-29-miqldap-to-sssd.md [hunk 1]
```diff
@@ -14,22 +14,24 @@ How does one use the *miqldap_to_sssd* conversion tool?
...
+**A brief Why before The How**
 =====================================================================
 
 Why did the *miqldap_to_sssd* conversion tool come to be?
 
-ManageIQ (MiQ) is delivered with the built in LDAP client, *MiqLdap*. Maintaining our own LDAP
-client is unnecessary as there are multiple mechanisms built into the operating system that provide an interface to an LDAP directory.
+ManageIQ (MiQ) is delivered with the built in LDAP client, *MiqLdap*. There are multiple, more robust and
+customizable, mechanisms built into the operating system that provide an interface to an LDAP directory.
+The  *miqldap_to_sssd* conversion tool is intended to automate and simplify moving to them.
```
Comment (abellotti) [inline site/_posts/2017-09-29-miqldap-to-sssd.md]: to them is too generic,  we only move to the SSSD authentication mechanism.

File: site/_posts/2017-09-29-miqldap-to-sssd.md [hunk 2]
```diff
@@ -311,6 +312,22 @@ argument can be used.
...
+
+It is beyond the scope of this document to describe the extensive configuration options. It will be left to the user to research
+[**(SSSD)**](https://docs-old.fedoraproject.org/en-US/Fedora/16/html/System_Administrators_Guide/chap-SSSD_User_Guide-Introduction.html).
+
+One noteworthy configuration parameter is **ldap_default_authtok**
+The  *miqldap_to_sssd* conversion tool populates this parameter in the **SSSD** configuration file **/etc/sssd/sssd.conf**
+
+**ldap_default_authtok** is a string that represents the authentication token of the default bind DN. Currently only clear text passwords are supported.
+Although the **SSSD** configuration file **/etc/sssd/sssd.conf** has limited ownership flags, some may want to obfuscate this.
+To accomplish this one could investigate [sss_obfuscate(8)](https://linux.die.net/man/8/sss_obfuscate)
```
Comment (abellotti) [inline site/_posts/2017-09-29-miqldap-to-sssd.md]: one could leverage the .... tool.

File: site/_posts/2017-09-29-miqldap-to-sssd.md (hunk 1 above)
Comment (jvlcek) [reply site/_posts/2017-09-29-miqldap-to-sssd.md]: Good point! 👍 Thank you!

File: site/_posts/2017-09-29-miqldap-to-sssd.md (hunk 2 above)
Comment (jvlcek) [reply site/_posts/2017-09-29-miqldap-to-sssd.md]: ... and again Good point! 👍 Thank you!

**Comment by abellotti** [comment]
LGTM!! Thanks @jvlcek for the enhanced doc. 👍

**Comment by jvlcek** [comment]
Thank you @abellotti !!!

**Comment by jvlcek** [comment]
@cybette Can you merge this please?

**Pull request merged**